pants.toml

```diff
@@ -14,14 +14,14 @@ backend_packages.add = [
   "pants.backend.python.typecheck.mypy",
   "pants.backend.python.mixed_interpreter_constraints",
   "internal_plugins.releases",
-#  "toolchain.pants.auth",
-#  "toolchain.pants.buildsense",
-#  "toolchain.pants.common",
+  "toolchain.pants.auth",
+  "toolchain.pants.buildsense",
+  "toolchain.pants.common",
 ]
 
-#plugins = [
-#  "toolchain.pants.plugin==0.3.0",
-#]
+plugins = [
+  "toolchain.pants.plugin==0.3.0",
```

Comment:
you can update to 0.4.0 which supports python 3.9. 
see https://github.com/pantsbuild/pants/pull/11471

Reply:
I don't think that will fully solve the problem - the issue iiuc is the dependency on pantsbuild.pants, which has no Py39 wheel to install. But this is a good change made regardless, so sg.